backend/api/metrics_views.py

```
@@ -19,6 +19,7 @@ class ActiveValidatorsView(APIView):
     Get active validators based on their first uptime contribution.
     Returns data points showing validator activation over time with continuous dates.
     """
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)